pkg/frontend/admin_hive_syncset_get.go

```diff
@@ -62,7 +55,13 @@ func (f *frontend) _getAdminHiveSyncSet(ctx context.Context, namespace string, s
 		return nil, api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidRequestContent, "", "syncsetname cannot be null")
 	}
 
-	ss, err := f.hiveSyncSetManager.Get(ctx, namespace, syncsetname, ssType)
+	var ss interface{}
```

Comment:
I think this works however since this is a refactor, I think we can use a real interface here? 

Something like

```
type Syncset interface {
  ToString() string
  GetType() string // selectorsyncset or syncset
}
```